SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -196,7 +196,7 @@ struct FSpatialLaunchConfigDescription
 	TArray<FWorkerTypeLaunchSection> ServerWorkers_DEPRECATED;
```

Comment:
How much do we care about backwards compatibility? Obviously, I'm changing a lot of things here, and I don't see a way to cleanly migrate existing configs.

Reply:
Don't have any context on our policies with _DEPRECATED things, @ImprobableNic  ?